src/constants/storage-type.ts

```diff
@@ -0,0 +1,6 @@
+enum StorageType {
+  // eslint-disable-next-line no-unused-vars
+  redis = "redis",
+}
+
+export { StorageType };
```

Comment:
```suggestion
export const StorageType = {
  redis: "redis",
} as const;
```
What do you think about using this ?

Reply:
removes the eslint comment so great.

Reply:
it's now returning as const and not a type so everywhere we would need to do `typeof StorageType `